### PR TITLE
modify merge configfile and custom config

### PR DESCRIPTION
**Pull request by rei-m**
Hi! I'm try to use [gulp-tslint](https://www.npmjs.com/package/gulp-tslint) and supply as a parameter by setting the configuration property.

ex )
```
gulp.task('tslint-json', function(){
    return gulp.src('invalid.ts')
        .pipe(tslint({
            configuration: {
              rules: {
                "class-name": true,
                // ... 
              }
            }
        }))
        .pipe(tslint.report('prose'));;
});
```

But property I set is ignored by loader. 
Please check my pr. 

Best regards.

### CODE REVIEW

**Comment by yangmillstheory** [comment]
See panuhorsmalahti/gulp-tslint#49.

File: index.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ function RcLoader(name, userConfig, finderConfig) {
           respond(err, configFile);
         });
       }
-      configFile = _.merge(config, configFile || {});
```
Comment (yangmillstheory) [inline index.js]: Tried this locally, causes [other tests to fail](https://travis-ci.org/spalger/rcloader/builds/69603164). :disappointed:

**Comment by yangmillstheory** [comment]
@rei-m could close this PR? gulp-tslint [upgraded to 4.3.0](https://github.com/panuhorsmalahti/gulp-tslint/commit/9c949f83931802e3b94d33701cda7b02804be33b) which resolves this issue.

**Comment by spalger** [comment]
:+1:

**Comment by rei-m** [comment]
oh... Sorry, I couldn't notice your comment. Thanks a lot.

**Pull request closed without merging**